### PR TITLE
[Hub Generated] Review request for Microsoft.CostManagement to add version stable/2020-06-01

### PR DESCRIPTION
**Pull request by paornela**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/paornela/azure-rest-api-specs/dev-cost-management-Microsoft.CostManagement-2020-06-01...master/).

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by chiragg4u** [comment]
Please go through the contribution checklist added as part of the review and mark them completed. This is a requirement before review can be continued.

**Comment by chiragg4u** [comment]
Looks like the change is to split the file and no new API introduced/change. Removing ARMFeedback required tag.

File: specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.exports.json [hunk 1]
```diff
@@ -0,0 +1,1140 @@
...
+					"enum": [
+						"MonthToDate",
+						"BillingMonthToDate",
+						"TheLastMonth",
+            "TheLastBillingMonth",
+            "TheLast7Days",
+						"WeekToDate",
+						"Custom"
+					],
+					"x-ms-enum": {
```
Comment (markcowl) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.exports.json:944]: This appears to be duplicated between the specs, per the lintdiff failure.

**Review by markcowl — CHANGES_REQUESTED** [review]

**Review by markcowl — APPROVED** [review]
Looked at6 all the listed breaking changes, and all represent the breaking change tool not recognizing the api moved to the new file, with the exception of the 'apiVersion20200601Parameter' which is functionally equivalent to the apiVersion parameyter

**Pull request merged**